### ISSUE TITLE
pettersen et al. replication

### ISSUE THREAD

**Issue by charlesdgburns** [charlesdgburns/GridModels#2]
Still struggling to replicate clean grid cells across the whole population in the final 256-neuron layer.

Might be due to sampling of points or room size, but it is neither obvious to me which or why.

I don't think I'm doing the loss function wrong, but the encoding loss does seem to increase its variability (which looks like some sort of overfitting artefact).

Peter suggested running on all open field sessions, for more data to sample from, so I'll give that a try.
